index.html

```diff
@@ -5,42 +5,59 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Hello World</title>
 
-    <link rel="stylesheet" href="timers.css">
+    <link rel="stylesheet" href="index.css">
   </head>
   <body>
     <div id="root"></div>
       
-      <!-- KINDA UNSUPPORTED CSS ANIMATION -->
```

Comment:
does this mean the css animations are supported now?

Reply:
That's actually something I forgot to remove from when I tried to do the animation the first time. I'll remove that comment.

Reply:
you did remove it, i just wanted to make sure the removal meant it's supported now and it wasn't removed by accident

Reply:
Yeah it should be fine. I'm not using CSS animation (@keyframe) for it. I'm using the transform: rotate() approach which I believe is very well supported by all modern browsers.